### PR TITLE
Refactor/core project name pom files

### PR DESCRIPTION
**Pull request by wilderrodrigues**
This PR replaces CloudStack with Cosmic in the Maven module names.

### CODE REVIEW

**Comment by wilderrodrigues** [comment]
Ping @borisroman @miguelaferreira @bheuvel @remibergsma 

Tests:

* Build Cosmic
* Deploy DB
* Build RPMs
* Install agen on KVM
* Deploy DC
* Deploy isolated network and VM

The other integration tests are still running.

**Comment by wilderrodrigues** [comment]
Ping @borisroman @miguelaferreira 

Fixed the jira issue on the clint/pom.xml

File: plugins/user-authenticators/ldap/test/resources/cosmic.cloud.ldif [hunk 1]
```diff
@@ -16,13 +16,13 @@
 # under the License.
```
Comment (borisroman) [inline plugins/user-authenticators/ldap/test/resources/cosmic.cloud.ldif]: This file @wilderrodrigues ? What does it do?

File: pom.xml [hunk 2]
```diff
@@ -605,7 +605,7 @@
               <exclude>systemvm/patches/debian/config/var/www/html/latest/.htaccess</exclude>
               <exclude>systemvm/patches/debian/vpn/etc/ipsec.d/l2tp.conf</exclude>
               <exclude>tools/transifex/.tx/config</exclude>
-              <exclude>tools/logo/apache_cloudstack.png</exclude>
+              <exclude>tools/logo/apache_cosmic.png</exclude>
```
Comment (borisroman) [inline pom.xml]: Is it already there?

Comment (wilderrodrigues) [inline pom.xml]: No, have to change it as well.

File: plugins/user-authenticators/ldap/test/resources/cosmic.cloud.ldif (hunk 1 above)
Comment (wilderrodrigues) [inline plugins/user-authenticators/ldap/test/resources/cosmic.cloud.ldif]: The intention was to have it being used by the LDAP unit tests, but it is not the case it.

image

**Comment by wilderrodrigues** [comment]
Ping @miguelaferreira @remibergsma @borisroman 

Applied change picked by @borisroman in his review. We need this in in order to move to the next step.

Tests for VM life cycle

```
Test advanced zone virtual router ... === TestName: test_advZoneVirtualRouter | Status : SUCCESS ===
ok
Test Deploy Virtual Machine ... === TestName: test_deploy_vm | Status : SUCCESS ===
ok
Test Multiple Deploy Virtual Machine ... === TestName: test_deploy_vm_multiple | Status : SUCCESS ===
ok
Test Stop Virtual Machine ... === TestName: test_01_stop_vm | Status : SUCCESS ===
ok
Test Start Virtual Machine ... === TestName: test_02_start_vm | Status : SUCCESS ===
ok
Test Reboot Virtual Machine ... === TestName: test_03_reboot_vm | Status : SUCCESS ===
ok
Test destroy Virtual Machine ... === TestName: test_06_destroy_vm | Status : SUCCESS ===
ok
Test recover Virtual Machine ... === TestName: test_07_restore_vm | Status : SUCCESS ===
ok
Test migrate VM ... SKIP: At least two hosts should be present in the zone for migration
Test destroy(expunge) Virtual Machine ... === TestName: test_09_expunge_vm | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 10 tests in 1609.016s

OK (SKIP=1)
(END)
```

**Pull request merged**